### PR TITLE
Add phiwings99

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: data/mods/ssb/items.ts [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+	// phiwings99
+	boatiumz: {
+		name: "Boatium Z",
+		isNonstandard: "Custom",
+		onTakeItem: false,
+		zMove: "Ghost of 1v1 Past",
+		zMoveFrom: "Destiny Bond",
+		itemUser: ["Froslass"],
+		gen: 8,
+		desc: "If held by a Froslass with Drain Punch, it can use Ghost of 1v1 Past.",
```
Comment (ghost) [inline data/mods/ssb/items.ts]: Drain Punch?

Comment (KrisXV) [reply data/mods/ssb/items.ts]: i literally typed this manually, im not sure how i did that

File: data/mods/ssb/moves.ts [hunk 2]
```diff
@@ -196,6 +196,37 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+		flags: {},
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Imprison', target);
+			this.add('-anim', source, 'Mean Look', target);
+			this.add('-anim', source, 'Transform', target);
+		},
+		volatileStatus: 'imprison',
```
Comment (ghost) [inline data/mods/ssb/moves.ts]: You should be applying thhe imprison to yourself or else it doesnt work

Comment (KrisXV) [reply data/mods/ssb/moves.ts]: O

**Pull request merged**